### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by mmm1k4** [BetmanSmall/TowerDefence_GDX#28]
Hi!
I came arround your repo when looking for the tower defense like games writen with libgdx. I am looking for to create a small project and I was wondering if the sourcecode is Apache 2, BSD or MIT so can be used. As there is no license in the project..

Can you update this? Else let me know if you are interested in publishing options.

Thanks

**Comment by BetmanSmall** [comment]
Thank you for your attention to my project. I am busy now, I will fix it within a week.

**Comment by BetmanSmall** [comment]
Hi! What project are you working on? I will be glad to share the code with you, however, I would also like to find fellow developers who are interested in making a tower defense game similar to WarCraft 3 Tropical Tower Wars. If you’re interested, write me back. Thanks!

**Comment by mmm1k4** [comment]
No, thank you. I was just interested in to getting know about the actual license of the public source code. Good luck with the game
